### ISSUE TITLE
Stop break line for ` ` character

### ISSUE THREAD

**Issue by IRediTOTO** [prettier/prettier#9380]
**Prettier 2.1.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEACAPAZzmGBLaAHSlVLPIrLABsBDTTAOVoFs4BeYMBgA2MoGCAJMBgB3HgDNqcAB6ppcgLQAnCBIC+-QTp4aAfNp0D0AEzwA3I8cE16TVhy69rNnSPE8xSgOwAOAAYAB3lFUJlZJUhqVABzWiClABY9VzdyPUMSdIFgQjR8rWycinQAIWoIMABrRjgxTFQgiEwYTmbWzABtAIBdDVQAeiySyjNLNNHSOwZmNk5uTD5iqYEPCTCFCNV1OITk1JXV0kzJqeAOmEwAOhYEgAp7y4AaVDwoUzkASlR2fVQ8kdjmQ8JJUPd3p95P8AqgAGRwt4fOQYVBJH6A4HGFRwGAAVxUJHQAGUAI542g4ipVapNFptC70gbVOAAT3a9OumGoeNiA2GAG4zsciliyBovqLgehBuYLCMpjK5QrRuMrECSjMHPNnEthSV1l4lABmABMwVkhyxpw1OWiqFN+sESssKrcMuwuAIUH0IGeIAgQXw0EwyFAlLUYgACpSEKGULRqGJaKzQ-6AEYqWg1XHEoLZ96xZAwFR4uD+uAsdNwUyfUwAGVoUFiFNicAAYhAVHcYPhm8gQLQ8TAIH6QAALGAsagAdXHeHgmHzYDgxLjC8sC9ZA8WY-e2BUMCjWdid2QkkT2H9ACtMLIylmczBiY56+84OfL+WQLfZMTCzIACKeIQPAn7UFeID5ioB4DumtDVtQY5BCo7wwDOeCmDA47IIE-ooS0cAzlmQQDihcAHhYH7+uSoFwMegbxoOmBKFAcA1jWY44uSeA4setCnrQ4GQdgLB4MWpbfpgAFwMBdHCd+MAIRhWE4Ugpr+iWtB4NQhYAMIQCwZ4oBRACsY54tgAAqCHxheEHfhYZYAJLIrAxJgKhQYAIIfMSMCsjICkaBoQA)
```sh
--parser babel
```

**Input:**
```jsx
 <section
                  className={css`
                     ${tw`flex flex-row`}
                  `}>
                  <div
                     className={css`
                        ${tw`w-780px flex flex-col gap-4`}
                     `}>
                     {" "}
                     <BlockNews post={posts[0]} />
                     <div
                        className={css`
                           ${tw`flex flex-row gap-4`}
                        `}>
                        {posts.map((post, index) => {
                           if (index > 0 && index < 4) {
                              return <SquareBlock post={post} key={post.slug} />;
                           }
                        })}
                     </div>
                  </div>
                  <div
                     className={css`
                        ${tw`w-320px`}
                     `}>
                     col 2
                  </div>
               </section>
```

**Output:**
```jsx
<section
  className={css`
    ${tw`flex flex-row`}
  `}
>
  <div
    className={css`
      ${tw`w-780px flex flex-col gap-4`}
    `}
  >
    {" "}
    <BlockNews post={posts[0]} />
    <div
      className={css`
        ${tw`flex flex-row gap-4`}
      `}
    >
      {posts.map((post, index) => {
        if (index > 0 && index < 4) {
          return <SquareBlock post={post} key={post.slug} />;
        }
      })}
    </div>
  </div>
  <div
    className={css`
      ${tw`w-320px`}
    `}
  >
    col 2
  </div>
</section>;

```

**Expected behavior:**
Hi, is there any config to stop break line for `` character? It should be like this, much more easy to read. 
```<div className={css` ${tw`w-780px flex flex-col gap-4`} `} >```
`

**Comment by thorn0** [comment]
Probably, Prettier should stop treating `css`-tagged template literals as embedded CSS when they're nested in JSX `className` attribute. Similar issue: https://github.com/prettier/prettier/issues/8363
Does anybody know of use cases that would be negatively affected by such a change?

**Comment by thorn0** [comment]
@IRediTOTO You are using [babel-plugin-tailwind-components](https://github.com/bradlc/babel-plugin-tailwind-components), right?

**Comment by IRediTOTO** [comment]
@thorn0  Yes, i am. Its fine, just wonder is there any config to stop that :D

**Comment by thorn0** [comment]
Prettier avoids configurability. If we figure out a way to improve this use case without making the behavior configurable, such a change most probably will be accepted. Otherwise, this will have to stay as is. That said, you can try using [`--embedded-language-formatting off`](https://prettier.io/blog/2020/08/24/2.1.0.html#add---embedded-language-formattingautooff-option-7875httpsgithubcomprettierprettierpull7875-by-bakkothttpsgithubcombakkot-8825httpsgithubcomprettierprettierpull8825-by-fiskerhttpsgithubcomfiskers) to fully disable formatting of embedded code like CSS-in-JS, but that's most probably not what you're looking for.

**Comment by IRediTOTO** [comment]
Year, I know. :D Just hurt the eyes. Thank you for reply me. 
![image](https://user-images.githubusercontent.com/15695252/95810124-74933200-0d3a-11eb-9efc-380a90147282.png)
![image](https://user-images.githubusercontent.com/15695252/95810204-a2787680-0d3a-11eb-848c-755c0c44b326.png)
